cmd/atlas/cmd/root.go

```diff
@@ -0,0 +1,25 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+// rootCmd represents the base command when called without any subcommands
+var rootCmd = &cobra.Command{
+	Use:   "atlas",
+	Short: "Work with any data source from the command line.",
+	Long:  `Work with any data source from the command line.`,
```

Comment:
?

Reply:
This is the output when doing help <command>
unlike Short -> general help page.